### PR TITLE
Math magicians: events

### PR DESCRIPTION
**Pull request by rega1237**
# Description

* In this project, I added the logic needed to make the Calculator component I developed in the previous project actually work.
* Create files calculate.js and operate.js
* Create the method in the class to calc and show in the calculator the result of the mathematical operation

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,30 +1,57 @@
...
-        <button type="button" className="leftCalc AC">A/C</button>
-        <button type="button" className="leftCalc plusMinus">+/-</button>
-        <button type="button" className="leftCalc reminder">%</button>
-        <button type="button" className="rightCalc div">&divide;</button>
-        <button type="button" className="rightCalc mult">X</button>
-        <button type="button" className="rightCalc sub">-</button>
-        <button type="button" className="rightCalc plus">+</button>
-        <button type="button" className="rightCalc equal">=</button>
+        <button type="button" className="leftCalc one" onClick={this.calc}>1</button>
+        <button type="button" className="leftCalc two" onClick={this.calc}>2</button>
```
Comment (KDlamini) [inline src/components/Calculator.js:38]: Please add the dot (`.` ) button that allows users to handle decimals.

image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @rega1237 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (rega1237) [reply src/components/Calculator.js:38]: @KDlamini  Oh I miss that, Thanks I will make the change

**Review by mateo951 — APPROVED** [review]
### Hi @rega1237, 

Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**